src/server/gameEngine/gameEngine.lib.ts

```diff
@@ -0,0 +1,70 @@
+import { INITIAL_CHESS_BOARD_FEN_STRING } from '../../chess/serialization/fen/fen.constants'
+import { createId } from '../../utils/uuid'
+import {
+	EngineChessColor,
+	EngineChessGame,
+	EngineChessGameStatus,
+	EngineChessPlayer,
+} from './store/store.models'
```

Comment:
Maybe use ’import type’ or 'import { type xxx }' here and elsewhere.

Reply:
Nice call. I agree that it would be clearer to differentiate between code and the types like this. My autoimports don't add the `type` prefix automatically, so maybe I'll setup a linter for it.

Reply:
Great comments by the way!